bin/writr

```diff
@@ -9,5 +9,5 @@ const writr = new Writr;
 writr.parseCLI(process);
 writr.runCLI().then(() => {
     process.exit();
-    log.info("Done!");
-});
\ No newline at end of file
+    console.info("Done!");
```

Comment:
Why did we remove the log class? I would like to keep that centralized in a class so we can control it.

Reply:
I did the change here because in this js file we don't import the log class to use the log method.